sys-libs/libcpuid/metadata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM 'http://www.gentoo.org/dtd/metadata.dtd'>
+<pkgmetadata>
+<maintainer type='person'>
```

Comment:
Repoman doesn't complain about it, but I've intended everything between `<pkgmetadata>` tags.